gsg-kit.md

```diff
@@ -229,35 +212,37 @@ The sample project now has Explorer HAT drivers installed and is ready for code
 1. On the Pi, generate code for the server app by running these commands:
 
    ```
-   cd ~/iot-lite/ #or path where you installed IoTivity
-   ./gen.sh
+   cd ~/workspace/myexample (if not already there)
+   gen.sh
    ```
 
 
-   This script starts from a default JSON configuration file that can be edited to specify the capabilities of your actual device. The script calls the DeviceBuilder app to generate source code for the server app you’ll run on your smart device (in this case, the Pi).
+   This script starts from a default JSON configuration file that can be edited to specify the capabilities of your actual device. The script calls the DeviceBuilder app to generate source code for the server app you’ll run on your smart device (in this case, the Pi). NOTE: We run gen.sh from the Project-Scripts directory (it's on the search path) NOT ./gen.sh
 
 2. Build the server app with this command:
 
 
    ```
-   ./build.sh
+   build.sh
```

Comment:
are you sure that ./  is not needed? depends on the bash i think

Reply:
This is using my scripts which are on the path and there is no local copy, so yes.